fe/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -786,15 +785,14 @@ private void expandStar(Analyzer analyzer, TableName tblName) throws AnalysisExc
      * Expand "*" for a particular tuple descriptor by appending
      * refs for each column to selectListExprs.
      */
-    private void expandStar(TableName tblName, TupleDescriptor desc) throws AnalysisException {
+    private void expandStar(TableName tblName, TupleDescriptor desc) {
         for (Column col : desc.getTable().getBaseSchema()) {
-            if (col.getDataType() == PrimitiveType.HLL && !fromInsert) {
-                throw new AnalysisException(Type.OnlyMetricTypeErrorMsg);
-            }
-            if (col.getAggregationType() == AggregateType.BITMAP_UNION && !fromInsert) {
-                throw new AnalysisException(Type.OnlyMetricTypeErrorMsg);
+            PrimitiveType type = col.getDataType();
```

Comment:
I don't think this change is very suitable.
1. I think the behavior of `select *` and `select a specific column` should be consistent.
2. The current modification method may have problems when inserting into select *, or when sub-queries, such as select hll_union (hll_col) from (select *);

I think we can do it in backend. When encountering HLL type or Bitmap type, ResultSink can send null directly.

Reply:
If we handle in backend:
1. there is a performance issue, we still need to scan  the HLL or Bitmap column
2. we need to support return binary content for HLL or Bitmap, so we shouldn't send null directly in ResultSink, we must pass a config from FE to BE.

Reply:
> 1. there is a performance issue, we still need to scan  the HLL or Bitmap column

I think performance may not be very important in this case

> 2. we need to support return binary content for HLL or Bitmap, so we shouldn't send null directly in ResultSink, we must pass a config from FE to BE.

Maybe we can support this feature later, and wait until the situation is really needed? Or the user can get the encoded readable content through the hex or base64 function